vignettes/develop.Rmd

```diff
@@ -0,0 +1,327 @@
+
+Then:
+
+```shell
+cmake -G "Unix Makefiles" -DWITH_UNIXODBC=1 -DWITH_SSL=/opt/homebrew/opt/openssl@3 -DODBC_INCLUDES=/opt/homebrew/Cellar/unixodbc/2.3.12/include -DODBC_LIB_DIR=/opt/homebrew/Cellar/unixodbc/2.3.12/lib
+```
+
+Towards the end of log outputs, you should see:
+
+```
+-- OpenSSL library: /opt/homebrew/opt/openssl@3/lib/libssl.dylib
+```
+
+or similar. If you see "not found," you will need to [troubleshoot your SSL library path](https://dev.mysql.com/doc/refman/8.0/en/source-ssl-library-configuration.html). 
+
+Finally:
+
+```
+make
+
+sudo make install
+```
+
+For more information on the above, see [MySQL's docs](https://dev.mysql.com/doc/connector-odbc/en/connector-odbc-installation-source-unix.html) and the arguments noted on the [installation documentation](https://dev.mysql.com/doc/connector-odbc/en/connector-odbc-installation-source-unix.html); you will likely need to set additional arguments.
+
+At the `make` step, you may see an error related to missing libraries.  If so, you'll need to (re)install the noted library with brew, clear the results of `cmake` from the source directory (I used `usethis:::git_clean()` to do so), and start again from the `cmake` step.
+
+```
+is a symlink belonging to openssl@3. You can unlink it:
+  brew unlink openssl@3
+
+To force the link and overwrite all conflicting files:
+  brew link --overwrite openssl@1.1
+```
+
+Once the driver is correctly installed, find the sample `.ini` files linked in the `sudo make install` output and add them to your configuration files noted in `odbcinst -j`. Remove the `DATABASE` key entry and set:
+
+```
+UID             = root
+PASSWORD        =
+```
+
+After running `brew services start mysql` if needed, and confirming that the database is running with `brew services info mysql`, you should be able to:
+
+```
+library(odbc)
+dbConnect(odbc(), "myodbc8w")
+```
+
+Where `"myodbc8w"` is substituted with the DSN you've configured.
+
+## SQL Server test setup
+
+To run Microsoft SQL Server on **aarch64 (e.g. M1 or M2) MacOS**, you will need: 
+
+* Docker 4.16 or higher
+* MacOS 13 Ventura (or higher)
+
+If needed, install Docker with:
+
+```shell
+brew install --cask docker
+```
+
+The Docker Desktop app provides a GUI to monitor deployed Docker containers and lives in `Docker.app > Show Package Contents > Contents > MacOS > Docker Desktop.app`.
+
+To [install the SQL Server ODBC driver and (optional) command line tool](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15), use:
+
+```shell
+brew tap microsoft/mssql-release https://github.com/Microsoft/homebrew-mssql-release
+brew install microsoft/mssql-release/msodbcsql18 microsoft/mssql-release/mssql-tools18
+```
+
+The `odbc.ini` entry should look something like:
+
+```ini
+[MicrosoftSQLServer]
+driver = ODBC Driver 18 for SQL Server
+Server = 127.0.0.1
+port = 1433
+Encrypt = no
+```
+
+In  `odbcinst.ini`:
```

Comment:
Unlike other drivers this will be updated automatically.

Reply:
[53b752c](https://github.com/r-dbi/odbc/pull/670/commits/53b752ca6bfd09bf7b9b30e78f817111a2caa58f).